include/z64cutscene.h

```diff
@@ -97,11 +97,11 @@ typedef struct {
 } CsCmdActorAction; // size = 0x30
 
 typedef enum {
-    CS_STATE_IDLE,
-    CS_STATE_SKIPPABLE_INIT,
-    CS_STATE_SKIPPABLE_EXEC,
-    CS_STATE_UNSKIPPABLE_INIT,
-    CS_STATE_UNSKIPPABLE_EXEC
+    CS_STATE_IDLE,              // CS_STATE_IDLE
```

Comment:
Can you explain the changes here?

Reply:
I used the documentation Fig did for decomp to help me figure out how z_demo was working, I just forgot to remove the extra stuff, I'll revert this considering this documentation is an opened PR on decomp so it'll be merged here someday